lib/pyenv.rb

```diff
@@ -18,16 +18,18 @@ def initialize(build_wrapper)
 
     def setup!
       install!
+      update!
```

Comment:
Thanks. Apparently `array1 - array2` does the trick, put in `uniq` at the end just to avoid potential dupes.